setup.py

```diff
@@ -52,7 +55,7 @@ def read_version_py(path):
         code = compile(fp.read(), 'kiva._version', 'exec')
     context = {}
     exec(code, context)
-    return context['git_revision'], context['full_version']
+    return context['git_revision'], context['version']
```

Comment:
this and some similar changes are unnecessary because I ended up keeping `full_version` even though it is the same as `version` now.